sdcm/cluster.py

```diff
@@ -75,6 +75,7 @@
 # Test duration (min). Parameter used to keep instances produced by tests that
 # are supposed to run longer than 24 hours from being killed
 TEST_DURATION = 60
+COREDUMP_MAX_SIZE = 1024 * 1024 * 1024 * 5
```

Comment:
Let's add a comment before this line

  #MAX limit to split coredump file (5G)